index.js

```diff
@@ -18,30 +19,48 @@ var startTime = function() {
   });
 };
 
+var potentiallyMigrateDatabase = function() {
+  return new Promise(function (resolve, reject) {
+    postgrator.setConfig({
+      migrationDirectory: __dirname + '/migrations',
+      driver: 'pg',
+      connectionString: process.env.POSTGRESQL_URL
+    });
+
+    postgrator.migrate('201505271423', function(err, migrations) {
+      if (err) {
+        reject(err);
+      } else {
+        postgrator.endConnection(function() {
+          resolve(migrations);
+        });
+      }
+    });
+  });
+};
+
 var loop = function () {
   var starttime;
   startTime()
     .then(function (time) {
       starttime = time;
     })
-
+    .then(potentiallyMigrateDatabase)
+    .then(function(migrations) {
+      console.log('Sucessfully migrated', migrations);
+    })
     .then(couch2pg)
     .then(function () {
       console.log('Imported successfully at ' + Date());
-    }, function (err) {
-      console.log('Import errored at ' + Date());
```

Comment:
I think I see where the logging is handled instead and had commented there.

I'm going to delete this comment and the one previous.